### PR TITLE
Fix Attention Ref Num - QA feedback

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#13878

*Description of changes:*

- Fix Attention Ref Num from showing up for different Transfers
- Update error message on Review screen

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by owillborn** [comment]
Too many changes... Looks good to me

**Review by owillborn — APPROVED** [review]

**Comment by dimak1** [comment]
Converting to Draft as I need to make another update...

**Review by ChasBoucher — APPROVED** [review]
Looks Good

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 1]
```diff
@@ -279,24 +277,27 @@ export default defineComponent({
...
+        return localState.validate && isValidReview ? errorMsg : ''
       }),
       reviewConfirmText: computed((): string => {
         return localState.isReviewMode ? 'Register Changes and Pay' : 'Review and Confirm'
       }),
       reviewOwners: computed(() => {
         return getMhrTransferHomeOwners.value.filter(owner => owner.action !== ActionTypes.REMOVED)
       }),
-      attentionReference: getMhrTransferAttentionReference.value,
+      attentionReference: '',
```
Comment (cameron-eyds) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue:300]: I guessed right.
Is store also preserving data on a cancellation of a Transfer?

Comment (cameron-eyds) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:300]: Using the getter here is not a bad way to do it, the root of the bug is probably that store isn't clear on cancel (Or re initialized during a new tranfers, either or)

Comment (dimak1) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:300]: The store is cleared on transfer initiation, but the value in the field (on the page) was still displaying the old value.(**Edit**: that was the bug when the value was still in the field somehow). I tested this fix and the data is preserved in the store when navigating Back and Next, and also it is properly resetting on the new Transfer.

Comment (cameron-eyds) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:300]: Oh! How was it preserving the store data if it was cleared (properly) on Transfer initiation?

Comment (dimak1) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:300]: It is resetting to empty as expected on Transfer initiation. Also, it is preserved in store (even with the getter removed).

Comment (cameron-eyds) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:300]: I understand that is what it's doing now after de-coupling the local property from the getter.
I'm referring to the original bug. 

Correct me if i'm wrong, wasn't the number preserved in store which was why the component was rendering with a previous entered Number on a new transfer?

Comment (dimak1) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:300]: That's the thing - the store was empty but the field in UI was not.

Screen Shot 2022-11-08 at 12 11 36 PM

Comment (cameron-eyds) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:300]: Very interesting indeed! 
Ok, will resolve this if you are confident that the bug its not a failure to clear the store data when initializing a new Transfer. 
Thanks for the discussion!

**Review by cameron-eyds — APPROVED** [review]

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:300]: During Transfer init, we do reset the data to be empty (all good there). But it is not reflected on the UI. 
Thanks!

**Pull request merged**